### PR TITLE
runtime: clh: Enforce to call 'cleanupVM' for 'stopSandbox'

### PR DESCRIPTION
**Pull request by likebreath**
We should always cleanup the vm directory when doing `stopSandbox`,
while we are skipping the cleanup process on some error code paths when
using cloud-hypervisor driver.

Fixes: #1098

Signed-off-by: Bo Chen <chen.bo@intel.com>

### CODE REVIEW

**Review by bergwolf — APPROVED** [review]

File: src/runtime/virtcontainers/clh.go [hunk 1]
```diff
@@ -764,6 +764,13 @@ func (clh *cloudHypervisor) terminate() (err error) {
 		pidRunning = false
 	}
 
+	defer func() {
+		clh.Logger().WithField("PID", pid).Info("cleanup VM")
```
Comment (liubin) [inline src/runtime/virtcontainers/clh.go:768]: Hi @likebreath  nit: here and below, `.WithField("PID", pid)` is not needed.

**Review by egernst — APPROVED** [review]
LGTM -- would be good to address Bin's suggestion, too, @likebreath

**Comment by likebreath** [comment]
@liubin PTAL. Thanks!

**Pull request merged**